language/analyze.js

```diff
@@ -12,163 +12,294 @@
+ * @param {string} [options.language] The language of the text, e.g. "en".
+ * @param {function} callback The callback function.
  */
-'use strict';
+function analyzeSentiment (text, options, callback) {
+  var document = language.document({
+    content: text,
+    type: options.type,
+    language: options.language
+  });
 
-var google = require('googleapis');
+  var config = {
+    // Get more detailed results
+    verbose: true
+  };
 
-var languageScopes = ['https://www.googleapis.com/auth/cloud-platform'];
+  // See https://googlecloudplatform.github.io/google-cloud-node/#/docs/language/latest/language/document
+  document.detectSentiment(config, function (err, sentiment) {
+    if (err) {
+      return callback(err);
+    }
 
+    console.log('Found %s sentiment', sentiment.polarity >= 0 ? 'positive' : 'negative');
+    return callback(null, sentiment);
+  });
+}
+// [END analyze_sentiment]
+
+// [START analyze_sentiment_from_file]
 /**
- * Gets a client that is connected to the Google Cloud Natural Language API.
+ * Detect the sentiment in a text file that resides in Google Cloud Storage.
+ *
+ * @param {string} bucket The bucket where the file resides.
+ * @param {string} filename The name of the file to be analyzed.
+ * @param {object} [options] Optional configuration.
+ * @param {string} [options.language] The language of the text, e.g. "en".
+ * @param {string} [options.type] "text" or "html".
+ * @param {function} callback The callback function.
  */
-function getLanguageService (callback) {
-  google.auth.getApplicationDefault(function (err, authClient) {
+function analyzeSentimentFromFile (bucket, filename, options, callback) {
+  var document = language.document({
+    content: storage.bucket(bucket).file(filename),
+    type: options.type,
+    language: options.language
+  });
+
+  var config = {
+    // Get more detailed results
+    verbose: true
+  };
+
+  // See https://googlecloudplatform.github.io/google-cloud-node/#/docs/language/latest/language/document
+  document.detectSentiment(config, function (err, sentiment) {
     if (err) {
       return callback(err);
     }
 
-    // Depending on the environment that provides the default credentials
-    // (e.g. Compute Engine, App Engine), the credentials retrieved may
-    // require you to specify the scopes you need explicitly.
-    if (authClient.createScopedRequired && authClient.createScopedRequired()) {
-      authClient = authClient.createScoped(languageScopes);
+    console.log('Found %s sentiment', sentiment.polarity >= 0 ? 'positive' : 'negative');
+    return callback(null, sentiment);
+  });
+}
+// [END analyze_sentiment_from_file]
+
+// [START analyze_entities]
+/**
+ * Detect the entities from a block of text.
+ *
+ * @param {string} text The text to analyze.
+ * @param {object} [options] Optional configuration.
+ * @param {string} [options.language] The language of the text, e.g. "en".
+ * @param {string} [options.type] "text" or "html".
```

Comment:
Describe this param more? e.g. `The format of the text (either "text" or "html").`